src/operation/operation_model_interface.jl

```diff
@@ -0,0 +1,42 @@
+# Default implementations of getter/setter functions for OperationsModel.
```

Comment:
This file indicates that we would require specific fields for all subtypes of OperationsModel. Are you sure about this?

Reply:
I think we want to have those requirements but probably there is more here than what we want. A lot of these functions need to be eliminated in favor of interfaces directly with the optimization container.